lib/routes/gov/chongqing/sydwgkzp.ts

```diff
@@ -54,7 +54,7 @@ async function handler(ctx: Context): Promise<Data> {
     const items = await Promise.all(
         list.map((item) =>
             cache.tryGet(item.link, async () => {
-                const { data: response }: { data: any } = await got(item.link);
+                const { data: response } = await got(item.link);
                 const $ = load(response);
                 // 主题正文
                 item.description = $('div[class="trs_editor_view TRS_UEDITOR trs_paper_default trs_web"]').first().html();
```

Comment:
```suggestion
                item.description = $('.trs_editor_view').first().html();
```
Consider fixing the description as well